historical_surveillance/models.py

```diff
@@ -0,0 +1,83 @@
+from django.db import models
+
+# from django.utils import timezone
+DISTRICT_CHOICES = [('district 1', 'District 1'), ('district 2', 'District 2'),
+                    ('district 3', 'District 3'), ('district 4', 'District 4'),
+                    ('district 5', 'District 5'), ('district 6', 'District 6'),
+                    ('district 7', 'District 7'), ('district 8', 'District 8')]
+GRADE_CHOICES = [('grade k', 'Grade k'), ('grade 1', 'Grade 1'), ('grade 2', 'Grade 2'), ('grade 3', 'Grade 3'),
+                 ('grade 4', 'Grade 4'), ('grade 5', 'Grade 5'), ('grade 6', 'Grade 6'),
+                 ('form 1', 'Form 1'), ('form 2', 'Form 2'), ('form 3', 'Form 3'), ('form 4', 'Form 4'),
+                 ('form 5', 'Form 5')]
+CATEGORY_CHOICES = [('primary school', 'Primary School'),
+                    ('secondary school', 'Secondary School'),
+                    ('other', 'Other')]
+
+SEX_CHOICES = [('male', 'Male'),
+               ('female', 'Female')]
+
+
+# TO-DO
+# Enforce Foreign Key constraint on the schools and districts
+
+class District(models.Model):
+    created_at = models.DateField(auto_now_add=True)
+    created_by = models.CharField(max_length=255)
+    District_Code = models.CharField(max_length=50, blank=True)
```

Comment:
Also it seems that having unique district codes is important. If we add `unique=True` here it will enforce uniqueness in the form by default so we don't need to include those checks in the view.

Reply:
No. the District code will not be a UUID should be for just student enrollment. The Idea of a System generated Universal Unique Identifier is for easier future interoperability. So what they can do later is whatever Id they choose to use as the student Id. It can be linked to the UUID. And all data that relate to a particular system are tagged with the UUID. There is a little bit of complexity to it in terms of creating the models. As I clean the code, I will make the naming conventions uniform,